docs/src/man/getting_started.md

```diff
@@ -21,13 +21,30 @@ that it builds correctly.
 To install `LazySets`, use the following command inside Julia's REPL:
 
 ```julia
-Pkg.clone("https://github.com/JuliaReach/LazySets.jl")
+Pkg.add("LazySets")
 ```
-The dependencies of `LazySets`, such as
-[Expokit.jl](https://github.com/acroy/Expokit.jl)
--- which provides lazy matrix exponentiation routines -- are automatically
-installed through Julia's package manager.
-The full list of dependencies is specified in the `REQUIRE` file.
+or replace `add` by `clone` if you want to develop the code.
+The full list of dependencies (which are automatically installed) is specified
+in the `REQUIRE` file.
+
+
+### Building the package
+
+Use the following command from Julia's REPL:
+```
+julia> using LazySets
+```
+This should precompile the package and make it available afterward.
+
+
+## Optional dependencies
+
+Some optional dependencies, such as
```

Comment:
I think it would be nice to list *all* optional dependencies here (or the next page) with explanations; i've opened https://github.com/JuliaReach/LazySets.jl/issues/1308.

Reply:
I resolve this comment here and defer to #1308.